### ISSUE TITLE
Training scannet200 model reports APs a lot higher than the numbers in this repo.

### ISSUE THREAD

**Issue by xuanlinli17** [JonasSchult/Mask3D#33]
Hi, I've been training scannet200 model using the default setting in the repo. However after 200 epochs, the reported APs on the validation set are already a lot higher than the checkpoints linked in this repo (https://github.com/JonasSchult/Mask3D/blob/main/docs/detailed_scores/scannet200_val.txt).

The AP/AP50/AP25 I got are `0.428          0.578          0.654`. 

Checkpoint can be downloaded [here](https://drive.google.com/file/d/1uUh1GZhedGMwALRPSHKqQu3gSvjMjTQ3/view?usp=share_link)

**Comment by xuanlinli17** [comment]
I think I found the reason. I was training on `https://github.com/JonasSchult/Mask3D/blob/main/scripts/scannet200/scannet200_benchmark.sh`, and this script has a problem: `data.train_mode=train_validation` will cause the model to be training on both training set and validation set.

**Comment by JonasSchult** [comment]
Hi Xuanlin,

thanks for your interest in the code!

The benchmark script trains on the training + validation set and is meant for a submission to the official ScanNet test set. Consequently, the performance on the validation set is very high.

Best,
Jonas
